lib/beaker/host/unix/exec.rb

```diff
@@ -142,7 +142,7 @@ def delete_env_var key, val
   #  host.get_env_var('path')
   def get_env_var key
     key = key.to_s
-    exec(Beaker::Command.new("env | grep #{key}"), :accept_all_exit_codes => true).stdout.chomp
+    exec(Beaker::Command.new("env | grep ^#{key}="), :accept_all_exit_codes => true).stdout.chomp
```

Comment:
It may well be, but it seemed less complicated to just modify the regexp used.  I don't know what the difference is between env and printenv on the various platforms (Linux and Cygwin)